tools/build_defs/cc/cc_import.bzl

```diff
@@ -184,7 +184,14 @@ def _cc_import_impl(ctx):
         name = ctx.label.name,
     )
 
-    return [CcInfo(compilation_context = compilation_context, linking_context = linking_context)]
+    this_cc_info = CcInfo(compilation_context = compilation_context, linking_context = linking_context)
+    cc_infos = [this_cc_info]
+
+    for dep in ctx.attr.deps:
+        cc_infos.append(dep[CcInfo])
+    merged_cc_info = cc_common.merge_cc_infos(cc_infos = cc_infos)
+
+    return merged_cc_info
```

Comment:
I'm not sure if it's required, but we tend to return single providers as a list (so `[merged_cc_info]`).